test/e2e/bsl-mgmt/deletion.go

```diff
@@ -205,88 +205,99 @@ func BslDeletionTest(useVolumeSnapshots bool) {
 			})
 
 			if useVolumeSnapshots {
-				if veleroCfg.CloudProvider == Vsphere {
-					// TODO - remove after upload progress monitoring is implemented
+				if veleroCfg.HasVspherePlugin {
 					By("Waiting for vSphere uploads to complete", func() {
 						Expect(WaitForVSphereUploadCompletion(oneHourTimeout, time.Hour,
 							bslDeletionTestNs, 2)).To(Succeed())
 					})
-					By(fmt.Sprintf("Snapshot CR in backup %s should be created", backupName_1), func() {
+					By(fmt.Sprintf("Snapshot CR in backup %s should be created", backupName1), func() {
 						Expect(SnapshotCRsCountShouldBe(context.Background(), bslDeletionTestNs,
-							backupName_1, 1)).To(Succeed())
+							backupName1, 1)).To(Succeed())
 					})
-					By(fmt.Sprintf("Snapshot CR in backup %s should be created", backupName_2), func() {
+					By(fmt.Sprintf("Snapshot CR in backup %s should be created", backupName2), func() {
 						Expect(SnapshotCRsCountShouldBe(context.Background(), bslDeletionTestNs,
-							backupName_2, 1)).To(Succeed())
+							backupName2, 1)).To(Succeed())
 					})
 				}
 				if veleroCfg.CloudProvider != VanillaZFS {
 					var snapshotCheckPoint SnapshotCheckPoint
 					snapshotCheckPoint.NamespaceBackedUp = bslDeletionTestNs
-					By(fmt.Sprintf("Snapshot of bsl %s should be created in cloud object store", backupLocation_1), func() {
-						snapshotCheckPoint, err = GetSnapshotCheckPoint(*veleroCfg.ClientToInstallVelero, veleroCfg, 1, bslDeletionTestNs, backupName_1, []string{podName_1})
+					By(fmt.Sprintf("Snapshot of bsl %s should be created in cloud object store", backupLocation1), func() {
+						snapshotCheckPoint, err = GetSnapshotCheckPoint(
+							*veleroCfg.ClientToInstallVelero,
+							veleroCfg,
+							1,
+							bslDeletionTestNs,
+							backupName1,
+							[]string{podName1},
+						)
```

Comment:
Is the message in L234 correct?

Reply:
No. There are a lot of similar errors. 
I will update them in later PRs.